fmriprep/workflows/bold/base.py

```diff
@@ -404,10 +413,12 @@ def init_func_preproc_wf(bold_file):
 
         inputnode.inputs.bold_file = ref_file  # Replace reference w first echo
 
-        join_echos = pe.JoinNode(niu.IdentityInterface(fields=['bold_files']),
-                                 joinsource=('meepi_echos' if run_stc is True else 'boldbuffer'),
-                                 joinfield=['bold_files'],
-                                 name='join_echos')
+        join_echos = pe.JoinNode(
+            niu.IdentityInterface(fields=['bold_files', 'skullstripped_bold_files']),
+            joinsource=('meepi_echos' if run_stc is True else 'boldbuffer'),
+            joinfield=['bold_files'],
```

Comment:
Is there a reason `skullstripped_bold_files` isn't also a `joinfield` ?

Reply:
Isn't `joinfield` referring to the appropriate field in the `joinsource`?

Reply:
The [tutorial](https://miykael.github.io/nipype_tutorial/notebooks/basic_joinnodes.html) indicates that it's the input fields of the `JoinNode`. It also says that it defaults to all input fields, but with dynamic interfaces like `IdentityInterface`, it's probably safer to be explicit.

Might as well give it a try and let the CI run tonight:

```suggestion
            joinfield=['bold_files', 'skullstripped_bold_files'],
```

Reply:
So the graph right now is merging in all of the iterables from `meepi_echos` (let's say), but we also need it to merge all of the outputs from `skullstrip_bold_wf` ? Which is not itself an iterable, but is being passed each of the echos (since it's downstream from `meepi_echos`).

I'd _guess_ (but definitely don't know for sure), that we could use `skullstripped_bold_files` under the same `joinsource`, since that's where the iterable is originating. Would love to hear if folks have other, better suggestions, though !